src/app/client/src/app/modules/learn/components/course-consumption/assessment-player/assessment-player.component.ts

```diff
@@ -59,6 +59,7 @@ export class AssessmentPlayerComponent implements OnInit, OnDestroy {
   totalContents = 0;
   consumedContents = 0;
   layoutConfiguration;
+  isCourseCompletionPopupShown = false;
```

Comment:
onCourseCompleteClose() {
    this.showCourseCompleteMessage = false;
  }

This was not added here. But the same is added in course-player-component

Reply:
onCourseCompleteClose method is already available